content/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-template-v1.md

```diff
@@ -0,0 +1,906 @@
+---
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "PodTemplate"
+content_type: "api_reference"
+description: "PodTemplate 描述了一种创建预定义 pod 副本的模板。"
+title: "PodTemplate"
+weight: 2
+auto_generated: true
+---
+
+<!--
+api_metadata:
+  apiVersion: "v1"
+  import: "k8s.io/api/core/v1"
+  kind: "PodTemplate"
+content_type: "api_reference"
+description: "PodTemplate describes a template for creating copies of a predefined pod."
+title: "PodTemplate"
+weight: 2
+auto_generated: true
+-->
+
+<!--
+The file is auto-generated from the Go source code of the component using a generic
+[generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
+to generate the reference documentation, please read
+[Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
+To update the reference content, please follow the 
+[Contributing upstream](/docs/contribute/generate-ref-docs/contribute-upstream/)
+guide. You can file document formatting bugs against the
+[reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.
+-->
+
+`apiVersion: v1`
+
+`import "k8s.io/api/core/v1"`
+
+
+## PodTemplate {#PodTemplate}
+
+<!--
+PodTemplate describes a template for creating copies of a predefined pod.
+-->
+PodTemplate 描述了一种创建预定义 pod 副本的模板。
+
+<hr>
+
+- **apiVersion**: v1
+
+
+- **kind**: PodTemplate
+
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **template** (<a href="{{< ref "../workload-resources/pod-template-v1#PodTemplateSpec" >}}">PodTemplateSpec</a>)
+
+  <!--
+  Template defines the pods that will be created from this pod template. https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+  -->
+  Template 定义将从这个 pod template 创建的pod。 https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+
+
+## PodTemplateSpec {#PodTemplateSpec}
+
+<!--
+PodTemplateSpec describes the data a pod should have when created from a template
+-->
+PodTemplateSpec 描述了从模板创建pod时应具有的数据
+
+
+<hr>
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  <!--
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+  标准对象的元数据。 更多信息： https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+
+- **spec** (<a href="{{< ref "../workload-resources/pod-v1#PodSpec" >}}">PodSpec</a>)
```

Comment:
We need to translate this page first, or the build won't be success.

Reply:
Waiting for #34294.
- #34294

Reply:
OK.  This page's build relies on other pages

Reply:
The docs/reference/kubernetes-api/workload-resources/pod-v1.md also relies:
docs/reference/kubernetes-api/config-and-storage-resources/volume.md
docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-claim-v1.md

these two file is in #https://github.com/kubernetes/website/issues/34158

Reply:
Yes, I need more time to handle `config-and-storage-resources/volume.md` that is relatively big with nearly 6000 words.